style/style.css

```diff
@@ -0,0 +1,112 @@
+* {
+    color : black;
+    font-family: 'Roboto Mono', monospace;
+    font-weight : 500;
+}
+
+header{
+    background-color : #F2F2F2;
+}
+
+nav{
+    display : inline-block;
+}
+
+main{
+    margin : auto;
+    width : 55%;
+}
+
+p{
+    font-weight : 300;
+}
+
+h3{
+    margin-top : 50px;
+    text-align : center;
+}
+
+ul{
+    font-size : 14px;
+    display : inline-block;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+}
+
+a:link {
+    text-decoration: none;
+}
+
+a:visited {
+    text-decoration: none;
+}
```

Comment:
It is not necessary to indicate the 2 pseudo-selectors, simply selecting all the labels to, you can apply that property